Samples/SampleSharedLib/SampleSharedLib/SampleSharedLib.vcxproj.filters

```diff
@@ -29,8 +29,8 @@
     <ClCompile Include="FileHelper.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="dllload.cpp">
-      <Filter>Source Files</Filter>
-    </ClCompile>
```

Comment:
Was it removed on purpose?

Reply:
Yes, in fact I would even recommend that the dllload.cpp file should just be removed. This was originally used to be able to load our inbox Windows.AI.Machinelearning.dll from local folder as the app. However, now we have NuGet as a solution, we don't want to have this backdoor way of loading our inbox binaries in our sample.